admin_manual/configuration_files/encryption_details.rst

```diff
@@ -26,7 +26,7 @@ This is the default encryption mode in Nextcloud. With master key encryption ena
 Key type: public sharing key
 ----------------------------
 
-The public sharing key is used to secure files that have been publicly shared. The public sharing key is protected by a password that can be generated by the server administrator. The advantage of the public sharing key is that it is independent of the selected encryption mode so that Nextcloud is able to provide publicly shared files to outside parties.
+The public sharing key is used to secure files that have been publicly shared. The public sharing key is protected by a password that can be set by the server administrator. A default password is automatically generated once encryption is enabled. The advantage of the public sharing key is that it is independent of the selected encryption mode so that Nextcloud is able to provide publicly shared files to outside parties.
```

Comment:
```suggestion
The public sharing key is used to secure files that have been publicly shared. The public sharing key is protected by a password that can be set/changed by the server administrator. A default password is automatically generated once encryption is enabled. The advantage of the public sharing key is that it is independent of the selected encryption mode so that Nextcloud is able to provide publicly shared files to outside parties.
```

Is this true @artonge i.e When the default password is generated is can be changed by the administrator?

Reply:
I don't think so. Better remove it then.

Reply:
I can't find a way to do so by checking occ, the web UI settings, and also reading through the code. So I think not.

Reply:
So not possible, leave at the level of auto-generated then.